docs/sftt/sftt-setup-backend.md

```diff
@@ -0,0 +1,32 @@
+# Setting up the Backend
+This is a comprehensive overview of the steps required and everything needed to run the database locally. 
+
+### Downloads
```

Comment:
On these downloads, maybe write "Download and Install" instead of just "Download". I know it's pretty implicit but just for clarity's sake.